scripts/amd/gemm/tune_gemm.py

```diff
@@ -29,6 +29,7 @@ def get_full_tuning_space():
     # other values in the future
     num_stage_range = [0]
     waves_per_eu_range = [0]
+    matrix_instr_nonkdim_range = [16, 32]
```

Comment:
Does it make more sense to write out the inst like 32x32x8? 

It is more scalable if the hw team introduces two insts with a different K dim but same non-K dim.

Reply:
With Alek's mfma4x64, we are facing this problem. 
But for now, the kernel still takes one value as the mfma instruction size. I can extend it when it's necessary :)